.circleci/config.yml

```diff
@@ -31,16 +31,20 @@ jobs:
           command: oasis build
       - run:
           name: Test Hello World
-          working_directory: hello-world/service
-          command: oasis test
+          working_directory: hello-world
+          command: source ../.circleci/scripts/common.sh && \
+                     oasis-chain > oasis-chain.log & && \
+                     oasis test
       - run:
           name: Build Secret Ballot
           working_directory: ballot/service
           command: oasis build
       - run:
           name: Test Secret Ballot
-          working_directory: ballot/service
-          command: oasis test
+          working_directory: ballot
+          command: source ../.circleci/scripts/common.sh && \
+                     oasis-chain > oasis-chain.log & && \
+                     oasis test
```

Comment:
can you use a circleci command? ([example](https://github.com/oasislabs/oasis-rs/blob/master/.circleci/config.yml#L14), [reference](https://circleci.com/docs/2.0/configuration-reference/#commands-requires-version-21))

Reply:
Sure once the CI passes after inlining this 🤔